ULogViewer/Logs/DataSources/StandardOutputLogDataSourceProvider.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+namespace CarinaStudio.ULogViewer.Logs.DataSources
+{
+	/// <summary>
+	/// Implementation of <see cref="StandardOutputLogDataSourceProvider"/>.
```

Comment:
<see cref="ILogDataSourceProvider"/> for <see cref="StandardOutputLogDataSource"/>.